src/distrs/pois.jl

```diff
@@ -19,6 +19,14 @@ poislogpdf(λ::T, x::T) where {T <: Real} = xlogy(x, λ) - λ - loggamma(x + 1)
 
 poislogpdf(λ::Number, x::Number) = poislogpdf(promote(float(λ), x)...)
 
+poislogpdf(λ::Number, x::Number) = poislogpdf(promote(float(λ), x)...)
+
+function poislogcdf(λ::Number, x::Number)
+    return SpecialFunctions.loggamma(Int(x) + 1, float(λ)) - SpecialFunctions.logfactorial(Int(x))
+end
+
+poiscdf(λ::Number, x::Number) = exp(poislogcdf(λ, x))
```

Comment:
To me it seems it would be preferable to define
```suggestion
poiscdf(λ::Number, x::Number) = last(gamma_inc(floor(x + 1), λ))
poisccdf(λ::Number, x::Number) = first(gamma_inc(floor(x + 1), λ))
```

Reply:
Alternatively, one could also forward the precision setting to SpecialFunctions (https://juliamath.github.io/SpecialFunctions.jl/stable/functions_list/#SpecialFunctions.gamma_inc):
```suggestion
poiscdf(λ::Number, x::Number; precision=0) = last(gamma_inc(floor(x + 1), λ, precision))
poisccdf(λ::Number, x::Number; precision=0) = first(gamma_inc(floor(x + 1), λ, precision))
```

Reply:
This doesn't cost you anything, given that it computes both upper and first?

Reply:
Both results sum up to 1, so it should not cost anything. And I assume that otherwise there would be two separate functions in SpecialFunctions.

Reply:
> sum up to 1

Doh. Makes sense then:)